test/evergreen.yml

```diff
@@ -749,21 +751,141 @@ tasks:
             set -o errexit
             set -o verbose
 
-            if [ "Windows_NT" = "$OS" ]; then
-              ${test_env_vars|} python ./test/suite/run.py -v 2 ${smp_command|} 2>&1
-            elif [ "$(uname -s)" == "Darwin" ]; then
-              # Avoid /usr/bin/python, at least on macOS: with System Integrity
-              # Protection enabled, it ignores DYLD_LIBRARY_PATH and hence
-              # doesn't find the WiredTiger library in the local tree.
-              ${test_env_vars|} python ./test/suite/run.py -v 2 ${smp_command|} 2>&1
-            else # Ubuntu
-              # Change directory to where the local installed 'wt' binary is located, 
-              # to avoid libtool generated 'wt' script from being selected by run.py,
-              # which invokes relink_command that tries to changing to a non-existed 
-              # /data/mci/<uniq> directory, as 'make' is done by a separate 'compile' task.
-              cd .libs
-              ${python_test_env_vars|} python ../test/suite/run.py -v 2 ${smp_command|} 2>&1
-            fi
+            # Only Windows and OS X variants are expected to run this task
```

Comment:
I do not understand, how are we limiting this task to run on windows and osx here.

Reply:
The scope of this ticket (and recent works) is for Ubuntu build variant, and we plan to cover Windows & OS X variants later in a separate effort. Before that happens, I'd keep both Windows and OS X running the old `unit-test` task (the un-split Python suite tests).